fileutil/fileutil_posix.go

```diff
@@ -3,26 +3,35 @@
 package fileutil
 
 import (
+	"fmt"
 	"os"
-	"os/user"
+	"os/exec"
 	"strconv"
+	"strings"
 )
 
-// SecureFiles takes ownership of files, and then give them 0600 file permissions
+// SecureFiles makes the current user/group the owner of all files in
+// filepaths, with 0600 file permissions.
 func SecureFiles(filepaths []string) (err error) {
-	var currentUser *user.User
-	currentUser, err = user.Current()
+	// Use /usr/bin/id rather than user.Current due to https://bugzil.la/1566159
```

Comment:
Might be worth mentioning what we talked about re: potentially switching to CGO in the future in a comment here